op-batcher/Makefile

```diff
@@ -24,8 +24,14 @@ ifeq ($(shell uname),Darwin)
 	FUZZLDFLAGS := -ldflags=-extldflags=-Wl,-ld_classic
 endif
 
+# Check if TARGETARCH is set to arm64 and adjust compilers accordingly
+ifeq ($(TARGETARCH),arm64)
+    CC := aarch64-linux-gnu-gcc
+    CXX := aarch64-linux-gnu-g++
+endif
+
 op-batcher:
-	env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) go build -v $(LDFLAGS) -o ./bin/op-batcher ./cmd
+	env GO111MODULE=on GOOS=$(TARGETOS) GOARCH=$(TARGETARCH) CC=$(CC) CXX=$(CXX) CGO_ENABLED=0 go build -v $(LDFLAGS) -o ./bin/op-batcher ./cmd
```

Comment:
I think we don't have to disable CGO if CC/CXX are setup correctly.

Reply:
I wasn't able to get it to work without disabling CGO. I believe it is because this PR is using debian for the builder image and alpine for the target image, which use different standard c libs (glibc vs musl). Disabling CGO forces the binary to be self-contained instead of relying on the host's c library, thus avoiding the compatibility issue

Reply:
If we are disabling CGO then I think the CC and CXX are completely no-op, ignored. If we are going with the no-cgo solution then we shouldn't be installing cross-compilers or debian etc. at all.

Reply:
Good point. I will remove the CC and CXX options and use CGO_ENABLED=0 instead. This will allow us to use the smaller alpine images and enable the binaries to run on amd64 and arm64 machines.